src/julia-syntax.scm

```diff
@@ -972,9 +974,9 @@
 		       (values name '() 'Any)) ex)
       ((pattern-lambda (curly (-- name (-s)) . params)
 		       (values name params 'Any)) ex)
-      ((pattern-lambda (|<:| (-- name (-s)) super)
+      ((pattern-lambda (comparison (-- name (-s)) <: super)
```

Comment:
Needs to be `(-/ <:)` to make it match a literal `<:`. Otherwise `<:` is a pattern variable like other symbols.